doorpi/status/status_lib/history_snapshot.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+import os
+logger = logging.getLogger(__name__)
+logger.debug("%s loaded", __name__)
+
+DOORPI_SECTION = 'DoorPi'
+
+def get(*args, **kwargs):
+    try:
+	if len(kwargs['name']) == 0: kwargs['name'] = ['']
+        if len(kwargs['value']) == 0: kwargs['value'] = ['']
+
```

Comment:
ich hab das ein paar mal in den Quellcodes gesehen - die beiden if beginnen an unterschiedlichen stellen.
Das kann in Python zu Problemen führen - eventuell IDE prüfen oder wechseln